CHANGELOG.md

```diff
@@ -5,29 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Added
+
+- JWT key auto-refresh, can be enabled by providing `"auto_refresh": true` and `"ca_cert_name": "..."` in `set_jwt_issuer` proposal.
+  - Auto-refresh is currently only supported for providers following the OpenID Connect standard where keys are published under the `/.well-known/openid-configuration` path of the issuer URL.
+  - `ca_cert_name` refers to a certificate stored with a `set_ca_cert` proposal and is used to validate the TLS connection to the provider endpoint.
+
+### Changed
+
+- `update_ca_cert` proposal has been replaced by `set_ca_cert`/`remove_ca_cert`.
+
 ## [0.15.2]
```

Comment:
0.15.2 hasn't been released yet: https://github.com/microsoft/CCF/releases

Reply:
Ah, ok. I'll merge it, but I think it's a bit confusing to already put a (unqualified) version number there before a release happened.